.changeset/clever-phones-travel.md

```diff
@@ -0,0 +1,8 @@
+---
+'@finos/legend-graph': minor
+'@finos/legend-application-studio': minor
+'@finos/legend-application': minor
+'@finos/legend-application-query': minor
+---
+
+Add color indicator for `PackageableElementOption` elements depending on type ([#457](https://github.com/finos/legend-studio/issues/457)).
```

Comment:
These are mere modifications following the main change, so all bumps should be `patch` and we probably don't need any message.

```suggestion
```